rulesengine_client/tests/test_models.py

```diff
@@ -63,8 +64,12 @@ def test_init(self):
         self.assertEqual(
             rule.protocol,
             'http')
+        self.assertEqual(
+            rule.subdomain,
+            'www')
```

Comment:
Is it worth adding another test case for a multipart subdomain e.g., `foo.bar`?

Reply:
The subdomain match is needed only for subdomains that are canonicalized away by the cdx surt format. I'm not aware of any such multipart domains. Also, the complementary code in aitwb does not support a multipart subdomain.